### ISSUE TITLE
Recommended hyper-parameters for WISE in Llama-3-8B

### ISSUE THREAD

**Issue by wang-kee** [zjunlp/EasyEdit#501]
Hi,

Thanks for sharing the great work.

I wanted to ask about whether you have any recommended hyper-parameters for running WISE on LLAMA-3-8B? I have used the default hyper-parameters (which differs from the ones for LLAMA-2-7B), but the results are not as good as for LLAMA-2-7B.

I'd really appreciate it a lot if you could also share the results for WISE on LLAMA-3-8B with the recommended hyper-parameters.

**Comment by JizhanFang** [comment]
Hello, our findings on the performance of Wise on Llama-3-8B are largely consistent with the conclusions you provided. There is indeed a slight gap compared to Llama2-7B. Our experience with parameter tuning for Wise is as follows:

If the editing success rate (edit success/reliability) is not satisfactory, we recommend reducing the **act ratio** parameter. Additionally, adjusting the learning rate (**lr**) or increasing the number of iterations (**iter**) can help achieve better results.

If the generalization metrics (**generalization/portability**) are lacking, we suggest increasing the **mask ratio** parameter, combined with adjustments to the learning rate (**lr**) and iterations (**iter**) for improvement.

**Comment by JizhanFang** [comment]
Hello, this question is difficult to answer. In our actual operations, our approach is to select some data and conduct multiple trials to determine the chosen layer. But in fact, for different data, the layer position where the editing target appears can also vary. Therefore, if you want to precisely locate the position of the "object" in a triplet of editing data, you may need to use some LLM interpretability tools to handle it, such as what was done in this paper:
https://arxiv.org/abs/2503.16356

**Comment by JizhanFang** [comment]
Hello, this question is difficult to answer. In our actual operations, our approach is to select some data and conduct multiple trials to determine the chosen layer. But in fact, for different data, the layer position where the editing target appears can also vary. Therefore, if you want to precisely locate the position of the "object" in a triplet of editing data, you may need to use some LLM interpretability tools to handle it, such as what was done in this paper:
https://arxiv.org/abs/2503.16356
However, for WISE hyper parameter tuning, it is not necessary. You can simply experiment with the last few layers of the model (around layer 27) and select the one that performs the best.

**Comment by wang-kee** [comment]
Thanks a lot, that clarifies my question, please feel free to close the issue
